src/main/java/org/sunbird/user/service/UserUtilityServiceImpl.java

```diff
@@ -505,4 +503,86 @@ public void enrichUserInfo(List<String> fields, Map<String, Map<String, String>>
 			}
 		}
 	}
-}
+
+	@Override
+	public boolean isUserExist(String email) {
+		// request body
+		SunbirdApiRequest requestObj = new SunbirdApiRequest();
+		Map<String, Object> reqMap = new HashMap<>();
+		reqMap.put(Constants.FILTERS, new HashMap<String, Object>() {
+			{
+				put(Constants.EMAIL, email);
+			}
+		});
+		requestObj.setRequest(reqMap);
+
+		HashMap<String, String> headersValue = new HashMap<>();
+		headersValue.put(Constants.CONTENT_TYPE, Constants.APPLICATION_JSON);
+		headersValue.put(Constants.AUTHORIZATION, props.getSbApiKey());
+		try {
+			String url = props.getSbUrl() + props.getUserSearchEndPoint();
+
+			Map<String, Object> response = outboundRequestHandlerService.fetchResultUsingPost(url, requestObj,
+					headersValue);
+			if (response != null && "OK".equalsIgnoreCase((String) response.get(Constants.RESPONSE_CODE))) {
+				Map<String, Object> map = (Map<String, Object>) response.get(Constants.RESULT);
+				if (map.get(Constants.RESPONSE) != null) {
+					Map<String, Object> responseObj = (Map<String, Object>) map.get(Constants.RESPONSE);
+					int count = (int) responseObj.get(Constants.COUNT);
+					if (count == 0)
+						return false;
+					else
+						return true;
+				}
+			}
+		} catch (Exception e) {
+			throw new ApplicationLogicError("Sunbird Service ERROR: ", e);
+		}
+		return true;
+	}
+
+	@Override
+	public boolean validateIfUserContactAlreadyExists(int phone) {
```

Comment:
this is same duplicate method of isUserExist -- refer existing implementation code... it has one method to verify both email and phone instead of repeating the same code twice